src/auto_on_message.py

```diff
@@ -132,3 +133,60 @@ async def debug_action(self, message, *args, **kwargs):
     # def test(message):
     #     print(message.author.nick)
     #     return message_data(message.channel,message.author.nick)
+
+class discord_simulator(auto_on_message):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        config = self.bot.config
+        self.BLACKLIST = set(config['ds_blacklist']) # A set is used because it has O(1) lookup, and this will only be used for that purpose.
+        self.MIN_WORDS = config['ds_min_words'] 
+        self.IDEAL_WORDS = config['ds_ideal_words']
+        self.LIST_LENGTH = config['ds_list_length']
+        self.simulator_channel = self.bot.client.get_guild(config['server_id']).get_channel(config['ds_channel'])
```

Comment:
Can you verify that this actually works?

Reply:
already tested